lib/Swagger/src/ContentEntity/Api/EntityToEntityRelationshipsApi.php

```diff
@@ -178,6 +180,11 @@ public function createWithHttpInfo($entity_id, $other_entity_id, $relevance = nu
         } elseif (count($formParams) > 0) {
             $httpBody = $formParams; // for HTTP post (form)
         }
+        // this endpoint requires API key authentication
+        $apiKey = $this->apiClient->getApiKeyWithPrefix('AUTH-SECRET');
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "AUTH-SECRET" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)